nix/darwin-release.nix

```
@@ -19,18 +19,23 @@ let
 
 in pkgs.runCommand name {
     buildInputs = with pkgs.buildPackages; [
+      binutils
+      darwin.cctools
+      nix
       haskellBuildUtils.package
       zip
     ];
   } ''
-  mkdir -p $out release
+  mkdir -p $out $(pwd)/release
 
-  rewrite-libs release ${rtViewServiceExe}/bin/*
-
-  cd release
+  cd $(pwd)/release
   mkdir ./static
   cp -R ${staticDir}/* ./static/
 
+  cp ${rtViewServiceExe}/bin/* ./cardano-rt-view-service
```

```suggestion
  cp ${rtViewServiceExe}/bin/* .
```
should suffice. maybe there are more files that need copying (?).